### ISSUE TITLE
why 0x6D for bypassCRC?

### ISSUE THREAD

**Issue by keencave** [captainbeeheart/openfreestyle#2]
Any background why the value of the bypass flag is specific 0x6D? The code do different flag comparisions with that value.

**Comment by captainbeeheart** [comment]
All I got is that to me it seems to be debug flags if not set.
Bit 8 : All CRC check bypass if null (tst.b ; jge => test bit8)
Bit 7 : Poweroff bypass in case of error if null (func 0x00005D18)
Bit 6 : will enable/disable an ADC related function...

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by keencave** [comment]
Agree. Looking at function 0x54a8 the flag is masked with 0x1F and shifted left by 4. Then it is passed to function 0x5e5c. I guess that is calculating a fraction of 15 (or 6.6%). The code is hard to understand. I simply simulated it and checked the results. Any idea on that?
